target/linux/ath79/dts/qca9558_mikrotik_routerboard-962uigs-5hact2hnt.dts

```diff
@@ -0,0 +1,227 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mikrotik,routerboard-962uigs-5hact2hnt", "qca,qca9558";
+	model = "MikroTik hAP ac";
+
+	aliases {
+		led-boot = &led_user;
+		led-failsafe = &led_user;
+		led-upgrade = &led_user;
+		serial0 = &uart;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_user: user {
+			label = "green:user";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 20 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	ath10k-leds {
+		compatible = "gpio-leds";
+
+		wlan5g {
+			label = "green:wlan5g";
+			gpios = <&ath10k 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+	};
+
+	gpio-export {
+		compatible = "gpio-export";
+
+		gpio_usb_power {
+			gpio-export,name = "power:usb";
```

Comment:
I'd not use the colon-syntax here, but just something like power-usb or usb-power.

Reply:
This block was copied from `qca9558_mikrotik_routerboard-922uags-5hpacd.dts`. I agree with your syntax, however would it be preferable to have consistent labels across similar boards?

Reply:
Why do you resolve this block if you ask a question? That makes no sense at all and just looks like you want to hide open problems from reviewers.

Coming back to the subject: This train has been missed a long time ago. gpio-export names are inherently inconsistent. But if you prefer the ":" syntax, I will not stop you from using it. But be aware that one cannot change this name without migration, so choose wisely.

Reply:
Changed per your feedback, marked resolved. Then reconsidered, sought clarification and forgot to un-resolve. If I wanted to hide problems I would not ask any questions :)

I don't have strong opinions on syntax here, seems we are both happy with `usb-power`.